.github/workflows/ci-image-scanning.yaml

```diff
@@ -31,9 +31,9 @@ jobs:
           - karmada-metrics-adapter
     steps:
       - name: checkout code
-        uses: actions/checkout@v4
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7

```

Comment:
We have [dependabot](https://github.com/karmada-io/karmada/blob/master/.github/dependabot.yml) bumps these github action regularly, would this (changing to commit version) block dependabot?

Reply:
@liangyuanpeng PTAL

Reply:
> We have [dependabot](https://github.com/karmada-io/karmada/blob/master/.github/dependabot.yml) bumps these github action regularly, would this (changing to commit version) block dependabot?

No

Reply:
> We have [dependabot](https://github.com/karmada-io/karmada/blob/master/.github/dependabot.yml) bumps these github action regularly, would this (changing to commit version) block dependabot?

Dependabot will also use the commit version the next time it updates the version.

Reply:
Good to know. Thanks for the confirmation.